README.md

```diff
@@ -24,10 +24,10 @@ PhoneGap plugin for managing application assets with javascript asset maps. Incl
 
 ### downloadFile()
 
-**wizAssets.downloadFile(String URL, String filePathToBeStoredWithFilename, Function success, Function fail);**
+**wizAssets.downloadFile(String URL, String URI, Function success, Function fail);**
```

Comment:
It was a sincere answer :)
I think this is really project-dependent and should not be limited by wizAssets. Users just have to know that they can use file path-like URI. Or am I missing something in your question?